### DISCUSSION TITLE
Possible to change authentication method?

### DISCUSSION THREAD

**Discussion by Dunrar** [firefly-iii/firefly-iii#6833] [Firefly III]
I have Firefly running behind a reverse proxy, Authelia and using REMOTE_USER header authentication. This works just as expected. But when testing my backups, I wanted to do so without having to setup the reverse proxy, Authelia and all that stuff. This is of course problematic, so my question is if there is a way to log into the REMOTE_USER account without having Authelia running. Thank you!

**Comment by JC5** [comment]
The backup should not have anything to do with the remote user authentication. What's your process?

**Reply by Dunrar** [reply]
My process is that I create a backup of the database and everything else and then migrate that over to another PC. The only difference on that PC is, that I have neither Authelia nor Nginx set up. I wanted to try and get the backups working on localhost only, just to see if all the data is still there. Everything seems to work, but not surprisingly I cannot access firefly as a user that was created by using an authentication proxy without having that authentication proxy setup as well.

My question is if I can access my new Firefly III instance as my old user without also having to get Authelia and Nginx up and running. Thanks!

**Reply by JC5** [reply]
Even remotely created users are normal users, so you can set `MAIL_MAILER=log` and then reset your password using the user interface. You should see the resulting email in the log files and using the instructions you can reset the password.

**Reply by Dunrar** [reply]
Okay, thanks, that kind of works. When trying that I get `Please enter an email address` for my main user (the Authelia username is not an email address). But it works for my second account, which uses a normal email address.

**Reply by Dunrar** [reply]
I guess this is the main problem. If the Authelia and Firefly passwords are also the same, I could probably simply log in. But the login form doesn't want me to enter something that is not an email address either.  Is there a way these rules could be relaxed if the user has been created via the REMOTE_USER header authentication?

**Reply by JC5** [reply]
What you can do, is inspect the field and change type=email to type=text. It might work then. Or, change the username in the DB.

**Reply by Dunrar** [reply]
Changing the type did not work. On login I get no matching credentials when using the correct password and no error at all when using an empty password, but I also do not get logged in. I'll try changing the username in the DB.

**Reply by Dunrar** [reply]
Okay, changing the username in the DB and then resetting the password via the UI worked. Thank you for your help and also for Firefly.
